<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Family Configuration</string>
  </property>
  <widget class="QListView" name="familyList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>150</width>
     <height>180</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="btnOk">
    <property name="text">
     <string>Ok</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="text">
     <string>Cancel</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>10</y>
     <width>150</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Family Name List</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddFamily">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>35</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelFamily">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>245</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QListView" name="listView_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>150</width>
     <height>180</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddVar">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>35</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelVar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>245</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="variableList">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>10</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Variable List</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>265</y>
     <width>310</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::HLine</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>35</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
